Documentation/platforms/arm/tiva/launchxl-cc1310/index.rst

```diff
@@ -0,0 +1,94 @@
+====================
+TI LaunchXL-CC1310
+====================
+
+SimpleLink Sub-1 GHz CC1310 wireless MCU LaunchPad development kit
+
+`TI LaunchXL-CC1310 <https://www.ti.com/tool/LAUNCHXL-CC1310>`_
```

Comment:
Integrate link in previous line. If you have a board picture, it would be good to add.